### PR TITLE
Ltac1: Simplify error message for not fully applied tactic

### LINKED ISSUE #9674: Error message for "The user-defined tactic "..." was not fully applied" should not sort variable arguments alphabetically
#### Description of the problem

Alphabetical sorting of variables is useless.  It makes it very hard to diagnose.  Consider this code:

```coq
Ltac stuff  q w e r t y u i o p a s d f g h j k l z x c v b n m := ().

Goal True.
  stuff () () () () () () () () () ().
  (* Error:
The user-defined tactic "stuff" was not fully applied: There are missing arguments for variables a,
s, d, f, g, h, j, k, l, z, x, c, v, b, n and m, arguments were provided for variables e, i, o, p, q,
r, t, u, w and y.
 *)
```

The missing arguments are fine, but the provided variables are sorted alphabetically, which is silly.

#### Coq Version

8.9

### PR DESCRIPTION
**Pull request by SkySkimmer**
Close #9674

### CODE REVIEW

**Comment by ppedrot** [comment]
I'm not so sure that the new variant is really that better. It should print the first non-applied variable or something, instead.

**Comment by SkySkimmer** [comment]
changed

File: plugins/ltac/tacinterp.ml [hunk 1]
```diff
@@ -1421,23 +1421,19 @@ and tactic_of_value ist vle =
...
-             Pp.str "an argument was provided for variable " ++
-               Pp.str (List.hd givenargs) ++ Pp.str "."
-          | _ ->
-             Pp.str "arguments were provided for variables " ++
-               pr_enum Pp.str givenargs ++ Pp.str ".")
+       Pp.(str tactic_nm ++ str " was not fully applied:" ++ spc() ++
+           str "There is a missing argument for variable" ++ spc() ++ Name.print (List.hd vars) ++
+           (if numargs > 1 then
+              spc() ++ str "and " ++ int (numargs - 1) ++
+              str (CString.plural (numargs-1) " more")
```
Comment (ppedrot) [inline plugins/ltac/tacinterp.ml:1428]: Wut? More is an adverb and can't be pluralized.

Comment (SkySkimmer) [reply plugins/ltac/tacinterp.ml]: I think at some point I had it as `more argument(s)` and didn't fix the plural when I removed `arguments`

fixed

**Comment by ppedrot** [comment]
@coqbot merge now

**Review by ppedrot — APPROVED** [review]

**Comment by ppedrot** [comment]
@coqbot merge now

**Pull request merged**